bundle/src/main/java/com/adobe/acs/commons/workflow/synthetic/impl/granite/SyntheticWorkflowSession.java

```diff
@@ -74,7 +76,84 @@ public SyntheticWorkflowSession(SyntheticWorkflowRunnerImpl workflowService, Ses
         this.backRoutes.add(new SyntheticRoute(true));
     }
 
-    @Override
+    public static SyntheticWorkflowSession createSyntheticWorkflowSession(SyntheticWorkflowRunnerImpl workflowService, Session session) {
+        return new SyntheticWorkflowSession(workflowService, session);
+    }
+
+    @Override
+    public Object invoke(Object proxy, Method method, Object[] args) throws Throwable {
+        String methodName = method.getName();
+        switch (methodName) {
```

Comment:
all methods which just throw UOE can be removed, as the handling is done already in the default branch.

Reply:
@kwin I updated how this is handled (removed it) so please let me know if that is sufficient.